### ISSUE TITLE
Control key is recognized as always down

### ISSUE THREAD

**Issue by zsugabubus** [i3/i3lock#260]
## I'm submitting a…

<pre>
[x] Bug
[ ] Feature Request
[ ] Other (Please describe in detail)
</pre>

## Current Behavior
Pressing <kbd>j</kbd>/<kbd>m</kbd>/<kbd>u</kbd>/<kbd>h</kbd> keys result in <kbd>Control</kbd>+<kbd>j</kbd>/<kbd>m</kbd>/<kbd>u</kbd>/<kbd>h</kbd>.

I can't see anything strange/useful in `--debug` log; `evtest` also confirms that I don't send <kbd>Control</kbd> key.

## Expected Behavior
Recognize control key only when the user has actually pressed down.

## Reproduction Instructions
1. Run:
```sh
i3lock # ...anything
```

## Environment
Output of `i3lock --version`:
<pre>
i3lock version 2.12 © 2010 Michael Stapelberg
</pre>
Arch Linux

**Comment by stapelberg** [comment]
Which keyboard layout are you using? How are you selecting it? What’s the output of `xmodmap -pke`?

**Comment by zsugabubus** [comment]
> Which keyboard layout are you using? How are you selecting it? 

I set `setxkbmap pl dvp compose:ralt` in an i3 `exec` but I tried setting different `setxkbmap`-s, replug the keyboard but I've experienced the same behavior. 

> What’s the output of xmodmap -pke?

I haven't modified anything so it should be the default.

---

As usual the issue gone away just after I reported it. I think updating `libxkbcommon` to v0.10.0 solved my issue. (I remember that `i3lock` worked with v0.8.4 so I guess v0.9.* introduced this behavior.)
